client/my-sites/upgrades/index.js

```diff
@@ -289,5 +290,14 @@ module.exports = function() {
 			controller.siteSelection,
 			upgradesController.checkout
 		);
+
+		page(
+			'/start-trial/:domain/:plan_name?',
```

Comment:
> Then plan_name won't be optional but I'm find with this.

It can't be really, as we need to know what plan they want

>So /start-trial/:plan_name/:domain ?

:+1: 

> Whitelisted, you mean removing the isEligible middleware?

No. This is something that we need to do on the server, if we want to.